subt_ign/launch/cloudsim_sim.ign

```diff
@@ -310,6 +310,12 @@
       <!-- This path is used by cloudsim, please do not change -->
       <path>/tmp/ign/logs</path>
       <overwrite>true</overwrite>
+      <record_topic>/logs/sdf</record_topic>
+      <record_topic>/world/<%= $worldName %>/changed_state</record_topic>
+      <record_topic>/world/<%= $worldName %>/dynamic_pose/info</record_topic>
```

Comment:
we should be able remove the default topics now that the behavior is to always record them

Reply:
Removed in 3ff6e0d9457ade9d547079ab584142c48d0fc022